### PR TITLE
Add missing -ms and -moz css properties

### PR DESCRIPTION
**Pull request by life777**
Adds new properties from issue list #50
There are 4 new properties:

1. `-moz-control-character-visibility` - I couldn't find any reference to this property in docs and specs. I found only link to bug Mozilla bug tracker where this property was added: https://bugzilla.mozilla.org/show_bug.cgi?id=947588. And w3c emails list with discussions about it: https://lists.w3.org/Archives/Public/www-style/2014Mar/0476.html
2. `-ms-grid-columns` and `-ms-grid-rows` - these two properties are from Grid spec version 0. As I understood they were implemented in IE 10: https://msdn.microsoft.com/en-us/ie/hh772052(v=vs.94). Also I added three new syntaxes: `track-group`, `track-list-v0`, and `track-minmax`. `track-list` exists in mdn/data, but there is a new version of it which isn't compatible we version 0 for `grid-columns` and `grid-rows` properties.
3. `-ms-hyphenate-limit-last` - syntax was taken from spec: https://www.w3.org/TR/css-text-4/#hyphenate-line-limits

Thanks

### CODE REVIEW

**Comment by life777** [comment]
@lahmatiy Should I change something in this PR or it should be added in MDN data? Thanks

File: data/patch.json [hunk 1]
```diff
@@ -644,6 +660,18 @@
...
+      "comment": "used by old grid-columns and grid-rows syntax v0",
+      "syntax": "[ '(' [ [ <string> ]* <track-minmax> [ <string> ]* ]+ ')' [ '[' <positive-integer> ']' ]? | <track-minmax> ]"
+    },
+    "track-list-v0": {
+      "comment": "used by old grid-columns and grid-rows syntax v0",
+      "syntax": "[ [ <string> ]* <track-group> [ <string> ]* ]+ | 'none'"
+    },
+    "track-minmax": {
+      "comment": "used by old grid-columns and grid-rows syntax v0",
+      "syntax": "'minmax(' <track-breadth> ',' <track-breadth> ')' | 'auto' | <track-breadth> | 'fit-content'"
```
Comment (lahmatiy) [inline data/patch.json]: No quotes needed. The syntax should be:
```
minmax( <track-breadth> , <track-breadth> ) | auto | <track-breadth> | fit-content
```

File: data/patch.json [hunk 2]
```diff
@@ -644,6 +660,18 @@
       "comment": "missed; not sure we should add it, but no others except `shape` is using it so it's ok for now; https://drafts.fxtf.org/css-masking-1/#funcdef-clip-rect",
       "syntax": "<length> | auto"
     },
+    "track-group": {
+      "comment": "used by old grid-columns and grid-rows syntax v0",
+      "syntax": "[ '(' [ [ <string> ]* <track-minmax> [ <string> ]* ]+ ')' [ '[' <positive-integer> ']' ]? | <track-minmax> ]"
+    },
+    "track-list-v0": {
+      "comment": "used by old grid-columns and grid-rows syntax v0",
+      "syntax": "[ [ <string> ]* <track-group> [ <string> ]* ]+ | 'none'"
```
Comment (lahmatiy) [inline data/patch.json]: No quotes, no extra groups:
```
[ <string>* <track-group> <string>* ]+ | none
```

File: data/patch.json [hunk 3]
```diff
@@ -644,6 +660,18 @@
       "comment": "missed; not sure we should add it, but no others except `shape` is using it so it's ok for now; https://drafts.fxtf.org/css-masking-1/#funcdef-clip-rect",
       "syntax": "<length> | auto"
     },
+    "track-group": {
+      "comment": "used by old grid-columns and grid-rows syntax v0",
+      "syntax": "[ '(' [ [ <string> ]* <track-minmax> [ <string> ]* ]+ ')' [ '[' <positive-integer> ']' ]? | <track-minmax> ]"
```
Comment (lahmatiy) [inline data/patch.json]: No extra groups:
```
'(' [ <string>* <track-minmax> <string>* ]+ ')' [ '[' <positive-integer> ']' ]? | <track-minmax>
```

**Comment by lahmatiy** [comment]
Sorry for delay in review. Some fixes needed (see above).

**Comment by lahmatiy** [comment]
Thank you!

**Pull request merged**